### DISCUSSION TITLE
Issue with pictures on the assembly pages?

### DISCUSSION THREAD

**Discussion by razidan** [alanbjohnston/CubeSatSim#299] [General]
Hi All,

I'm not sure if it's just me, but about half of the pictures on the assembly instructions don't fully come up. I tried different computers and different WIFI networks.
Is it an issue with the site itself?

**Comment by alanbjohnston** [comment]
Hi, I have noticed that GitHub is serving the image files slowly lately.  I think it is an issue with GitHub, as when I check later, the images load OK.  I could reduce the image sizes from full resolution - this would likely help the problem, but not fix it.  If I right click on the images and select "Load in New Tab" I see the images.  This might help.  I've used GitHub to host the wiki images for over a year now, but maybe I have to host the images myself going forward.

Alan

**Comment by razidan** [comment]
Thanks for the fast reply.
I did try to open it on a new tab, but it still loads only partway. The
same part that shows on the main page.

**Comment by alanbjohnston** [comment]
Are you trying to access the Beta wiki pages for the blue PCBs? Let me know if this problem persists - I'll re-upload the images.

**Comment by N2OA** [comment]
I haven't had any issues with chrome. I just verified that Edge, Firefox,
and Opera all load fine as well. Images only take about a half second or so
to load but I have 1Gb fiber to the house. I also verified on my work PC
VPN'd into a corporate network. No issue there either.

You can try clearing the cache, turn off any ad blockers and check the
security and privacy settings. You can try incognito too.

Dave N2OA

**Comment by razidan** [comment]
No. V1.2.

**Comment by razidan** [comment]
I'll try those tips!

**Comment by razidan** [comment]
I had my IT dept at the school also check.
Same issue on different computers, different networks and different browsers... Not all pictures are like that, but a good amount are.
Weird.

**Comment by alanbjohnston** [comment]
I fixed a few bad images in the v1.2 instructions.  I don't know why they broke - the images are still there on my server, and I just changed the link from http to https to get GitHub to fetch it again, and they appeared.

I have also made a PDF image of each wiki page and put a link at the top.  If this happens again, you should be able to download this PDF, which is hosted on my web server.

You can see all of them here: https://CubeSatSim.org/pages

If issues come up on the v1.3.2 beta wiki instructions, let me know and I'll do something similar there.

**Reply by razidan** [reply]
Thanks!
On my machine it fixed some but not all of the picture. But the PDF is a good solution.
